core/jobs.py

```diff
@@ -661,6 +661,8 @@ def _real_enqueue(
 
 
 class MapReduceJobPipeline(base_handler.PipelineBase):
+    # pylint: disable=8-space-indentation-in-docstring
```

Comment:
I don't understand -- why disable this (and the line below)?

Reply:
This is part of the one edge case where the function yields two things instead of one and my linter breaks in that case since for returns and yields, it checks for the parameter by matching to a "[return type]." string but there's no way of differentiating a wrap around line in the description of a return type that also ends in a period which means that it treats the "StoreMapReduceResults" as part of the description. I think I asked you about this and you recommended just disabling it for this case since it only occurs once.

Reply:
I've been looking online but there doesn't seem to be a way of disabling pylint for a singular line inside a multiline comment. Do you know of another way of doing this?